frontend/src/app/assess/[projectId]/fairness-bias/dataset-testing/page.tsx

```
@@ -257,147 +342,181 @@ const DatasetTestingPage = () => {
+                    <div className="flex flex-col gap-3 md:flex-row md:items-center md:justify-between">
+                      <p className="text-sm text-slate-500 dark:text-slate-400 flex items-center gap-2">
+                        <FileText className="w-4 h-4" />
+                        Data never leaves this workspace.
+                      </p>
+                      <div className="flex gap-3">
+                        {hasFile && (
+                          <button
+                            onClick={handleReset}
+                            className="inline-flex items-center gap-2 rounded-xl border border-slate-200 dark:border-gray-700 px-4 py-2 text-sm font-medium text-slate-600 dark:text-slate-300 hover:bg-slate-50 dark:hover:bg-gray-800"
+                          >
+                            <Trash2 className="w-4 h-4" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)